fixtures/parser_tests.js

```diff
@@ -385,6 +385,52 @@ function parserTests(parse: (file: string) => ParseResult, isTypescript = false)
       assertBlock2(itBlock, 2, 7, 4, 9, 'each test %p');
       assertNameInfo(itBlock, 'each test %p', 2, 33, 2, 44);
     });
+    
+    it('For the simplest it.each cases', () => {
```

Comment:
Seems like we are missing the corresponding `describe.each` test cases? Also might be worthy to add nested ".each" cases ...

Reply:
It added a similar describe.each suite.